openid-sharedsignals-framework-1_0.md

```diff
@@ -429,14 +427,104 @@ below MAY define certain members within a Complex Subject to be Critical. A SSF
 Receiver MUST discard any event that contains a Subject with a Critical member
 that it is unable to process.
 
-# Event Properties  {#properties}
+# Events {#events}
+
+## Security Event Token Profile {#set-profle}
+The Shared Signals Framework profiles the Security Event Token (SET)
+{{RFC8417}} specification by defining certain properties of SETs as described in this section.
+
+### Explicit Typing of SETs {#explicit-typing}
+SSF events MUST use explicit typing as defined in Section 2.3 of {{RFC8417}}.
+
+~~~ json
+{
+  "typ":"secevent+jwt",
+  "alg":"HS256"
+}
+~~~
+{: title="Explicitly Typed JOSE Header" #explicit-type-header}
+
+The purpose is defense against confusion with other JWTs, as described in
+Sections 4.5, 4.6 and 4.7 of {{RFC8417}}. While current Id Token {{OpenID.Core}}
+validators may not be using the "typ" header parameter, requiring it for SSF
+SETs guarantees a distinct value for future validators.
+
+### SSF Event Subject {#event-subjects}
+The primary Subject Member of SSF events is described in the "Subject Members" section ({{subject-ids}}). The JWT "sub" claim MUST NOT be present in any SET containing
+an SSF event.
+
+### Distinguishing SETs from other Kinds of JWTs
+Of particular concern is the possibility that SETs are confused for other kinds
+of JWTs. The Security Considerations section of {{RFC8417}} has several sub-sections
```

Comment:
This is incorrect. Section 4 talks about it https://datatracker.ietf.org/doc/html/rfc8417#section-4